### DISCUSSION TITLE
Possible to run headless remote on Linux?

### DISCUSSION THREAD

**Discussion by VasiliyS** [zed-industries/zed#10722] [Feature Requests]
Is there a way (or plans) to enable usage/connectivity to a remote, possibly local virtual, Linux machine? Something similar to VSCode's capability.

**Comment by SomeoneToIgnore** [comment]
There's some work happening on that front, but it uses collab as a source of remote projects, but at least it runs on Linux:

* https://github.com/zed-industries/zed/pull/9680
* https://github.com/zed-industries/zed/pull/10085
* https://github.com/zed-industries/zed/pull/10594

**Reply by SomeoneToIgnore** [reply]
See also https://github.com/zed-industries/zed/issues/5347 for general tracking issue for the feature.
Although so far there's no ssh-based approach drafted or planned.

**Comment by JosephTLyons** [comment]
The alpha preview for remote dev recently landed in version [0.145.1](https://zed.dev/releases/stable/0.145.1).
